packages/get-platform/src/getPlatform.ts

```diff
@@ -156,34 +156,33 @@ type GetOpenSSLVersionParams = {
  * On Linux, returns the OpenSSL version excluding the patch version, e.g. "1.1.x".
  * Reading the version from the libssl.so file is more reliable than reading it from the openssl binary.
  * This function never throws.
- *
- * TODO: we should probably validate the output of this function, and ensure it's contained in `supportedLibSSLVersions`
  */
 export async function getSSLVersion(args: GetOpenSSLVersionParams): Promise<GetOSResult['libssl'] | undefined> {
-  const libsslSpecificPaths = await match(args)
-    .with({ distro: 'musl' }, async () => {
+  const archFromUname = await getArchFromUname()
```

Comment:
Can we move it to where it's used so we don't call `uname` on platforms where we don't need it?

Reply:
I'd rather not: this was something we did before, but made it cumbersome dealing with all the non-`debian` pattern branches: they had useless `async` declarations and needed `Promise.resolve` before returning the result. Besides, The function that computes `uname` is guaranteed to not raise errors by design, is cheap to call and lies in a non-performance-critical block of Prisma